docs/di.rst

```diff
@@ -63,9 +63,9 @@ How to use DIContainerTrait
 
 .. php:trait: DIContainerTrait
 
-.. php:method: setDefaults($array, $strict = false)
+.. php:method: setDefaults($array)
```

Comment:
We have `    public function setDefaults(array $properties, bool $passively = false)`

Reply:
yeah, docs should be generated...